Documentation/gettingstarted/external-workloads.rst

```diff
@@ -25,9 +25,9 @@ Prerequisites
   (``identityAllocationMode`` set to ``crd``). This is the default
   for new installations.
 
-* External workloads must run a recent enough kernel (>= 4.17) for k8s
+* External workloads must run a recent enough kernel (>= 4.19.57) for k8s
```

Comment:
Why the kernel ver bump in a PR just changing an identifier?

Reply:
We are getting rid of the host-reachable services protocols (already deprecated), and in order for both (i.e., TCP and UDP) to work, at least 4.19.57 is needed.

Reply:
But why was it 4.17 before?

Reply:
TCP is 4.17+, UDP is 4.19.57+ but afaik we're not removing the option yet, right?